src/lib/ibcclient.ts

```diff
@@ -1176,7 +1199,18 @@ export class IbcClient {
       };
       msgs.push(msg);
     }
-    this.logger.debug('MsgRecvPacket(s)', { msgs });
+    this.logger.debug('MsgRecvPacket(s)', {
+      msgs: msgs.map((msg) =>
+        formatMessage(msg, (mutableMsg, encode) => {
```

Comment:
This seems reasonable - manually choosing the fields to shorten rather than trying to automate it... this lets us customize much more easily.

Also, the use of mutableMsg makes this much easier than an immutible transform (even though Will will grimmace, I think it is the right choice)

Reply:
Haha, I did!